include/af/opencv2.hpp

```diff
@@ -0,0 +1,160 @@
+#pragma once
+#include <vector>
+#include <opencv2/core/core.hpp>
+#include <arrayfire.h>
+
+namespace afcv
+{
+
+/*
+ * opencv -> arrayfire
+ */
+// conversion for cv::Mat
+af::array array(const std::vector<cv::Mat>& input, bool transpose = true);
+af::array array(const cv::Mat& input, bool transpose = true);
+
+/*
+ * arrayfire -> opencv
+ */
+// af::array conversion to cv::Mat
+cv::Mat toMat(const af::array& input, int type = CV_32F, bool transpose = true);
+
+
+//helper conversion function
+void copy_data(Mat& input, af::array& output, bool transpose) {
```

Comment:
I mean if you included this specific header into two or more cpp source files then you will encounter a multiple definition error because this is not a template function